tr/django_models/README.md

```diff
@@ -110,7 +110,6 @@ INSTALLED_APPS = (
 Şimdi `blog/models.py` dosyasını açalım ve içindeki her şeyi silip şu kodu yazalım:
 
 ```
-python
 from django.db import models
```

Comment:
The word python is meant to be there but should be on the end of the previous line so the syntax highlighter knows this is python.

so you will have

```
```python
```